local-wallet/public/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-16" />
```

Comment:
Is there a reason that we're switching to UTF16?

Reply:
From @carlfairclough's  PR: `utf-16 html encoding so we can show utf-16 characters`. 

[Source](https://github.com/dmihal/maker-devcon-burner-wallet/pull/15)

In terms of the rationale behind that, I will have to leave it up to him.